core/modules/entityreference/tests/entityreference.handlers.test

```diff
@@ -21,7 +21,7 @@ class EntityReferenceHandlersTestCase extends BackdropWebTestCase {
     }
   }
 
-  protected function assertReferencable($field, $tests, $handler_name) {
+  protected function assertReferenceable($field, $tests, $handler_name) {
```

Comment:
If we wanted to be really safe about this (contrib/custom modules might be using it), we should add a wrapper functio and a `@todo` item to remove in 2.0

Reply:
...oh, this is a protected function though. So perhaps not.